best-practices/require-ro-rootfs/bad-pod-false.yaml

```diff
@@ -7,4 +7,4 @@ spec:
   - name: busybox
     image: busybox:1.35
     securityContext:
-      readOnlyRootFilesystem: false
```

Comment:
If we're not materially changing code, we should not be submitting these sort of changes to a public repo.  It just leads to more noise when we merge from public upstream. Can this change be reverted?